src/fastoad/notebooks/02_explanation/04_OpenMDAO.ipynb

```diff
@@ -0,0 +1,1010 @@
+{
```

Comment:
<blockquote>Quoting <a href="https://openmdao.org/newdocs/versions/latest/features/model_visualization/n2_basics/n2_basics.html?highlight=n2%20chart" target="_blank">OpenMDAO's documentation</a>: <code>"</code>an N2 diagram, also known as an N-squared diagram, is a diagram in the shape of a matrix, representing functional or physical interfaces between system elements<code>"</code>.</blockquote><p> </p><blockquote>this diagram contains the name of the module (not the name of the subsystem, hence why naming when adding a subsystem is important)</blockquote><p>I do not understand. To me, these are the subsystem names, as instructed in the <code>add_subsytem </code>calls. And the "module" notion is not really defined here, AFAIK.</p><p> </p><blockquote>As a e<code><s>n</s></code>xample, we will</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Changed in dfe233f7, I'd like your input on the way it is written now; We can also delete the sentence.

Reply:
<blockquote>In essence, this diagram contains the name of the subsystem<code>s</code> (not the name of the subsystem class, hence why naming when adding a subsystem is important)</blockquote><p>In the parenthesis, I would simply put "(the provided name in the <code>add_subsystem</code> command)"</p><p> </p>

Reply:
Changed in 3a141cc1